### PR TITLE
Change QPad to qStudio

### PR DESCRIPTION
**Pull request by David2464**
-Summary
Changed documentation from showing connecting to clusters using QPad to using QStudio

-Additionally
In examplequeries.q in code/rdb and code/hdb... Within the functions the table names were hard coded to be trade whereas the tables are actually names "trades".

Although I have noticed that in the Master version of code (Not Release), these tables change again back to "trade" and "quote"

### CODE REVIEW

**Review by jonathonmcmurray — APPROVED** [review]

File: docs/10-connectingusingqStudio.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Connecting Using qStudio
+===============
+
+Open qStudio, right-click on Servers in the top left corner and select 'Add Server'.
```
Comment (zanbuchanan) [inline docs/10-connectingusingqStudio.md:4]:
```suggestion
Open qStudio, right-click on 'Servers' in the top left corner and select 'Add Server'.
```

File: docs/10-connectingusingqStudio.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+![Add Server Connection](workshop/graphics/Servers_qStudio1.png)
+
+You will then see a pop-up showing the server properties. This is where you will add the information from the connection string.
```
Comment (zanbuchanan) [inline docs/10-connectingusingqStudio.md:8]:
```suggestion
You will then see a pop-up showing the server properties. This is where you will add the information from the connection string (details below in the next step).
```

File: docs/10-connectingusingqStudio.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+![Server Properties](workshop/graphics/Server_Properties1.png)
+
+The generated connection string provides you with everything you need to fill this out, take this connection string for example;
```
Comment (zanbuchanan) [inline docs/10-connectingusingqStudio.md:12]:
```suggestion
Follow the step in our [Generate Connection String section](https://dataintellecttech.github.io/TorQ-Amazon-FinSpace-Starter-Pack/08-clusterconnectionstring/#generate-connection-string) to get a fresh connection string. The generated connection string provides you with everything you need to fill the boxes above. See the below connection string for an example:
```

**Review by zanbuchanan — COMMENTED** [review]
Just a few comments, looking good!

**Pull request merged**